server/src/main/java/org/opensearch/search/lookup/LeafSearchLookup.java

```diff
@@ -33,7 +33,6 @@
 package org.opensearch.search.lookup;
 
 import org.apache.lucene.index.LeafReaderContext;
-
```

Comment:
Super nit, could you please remove this change? (it is complicates filtering changes by file in Git)